src/genie/libs/parser/iosxe/show_cts.py

```diff
@@ -457,3 +458,641 @@ def filter_lines(raw_output, remove_lines):
+                    cts_rbacl_dict['cts_rbacl']['name'][rbacl_name]['aces'][rbacl_ace_index] = ace_group_dict
+                rbacl_ace_index = rbacl_ace_index + 1
+                continue
+        return cts_rbacl_dict
+
+
+# ====================================
+# Schema for:
+#  * 'show cts role-based permissions'
+# ====================================
+class ShowCtsRoleBasedPermissionsSchema(MetaParser):
+    """Schema for show cts role-based permissions."""
+
+    schema = {
+        "rbp_policies": {
+            int: {
+                Optional("policy_name"): str,
+                "action_policy": str,
+                "action_policy_group": str,
+                Optional("src_grp_id"): int,
+                Optional("src_grp_name"): str,
+                Optional("unknown_group"): str,
+                Optional("dst_group_id"): int,
+                Optional("dst_group_name"): str,
+                Optional("policy_groups"): {
+                    Optional(int): {
+                        Optional("policy_group"): str
+                    }
+                }
+            },
+            "monitor_dynamic": bool,
+            "monitor_configured": bool
+        }
+    }
+
+
+
+# ====================================
+# Parser for:
+#  * 'show cts role-based permissions'
+# ====================================
+class ShowCtsRoleBasedPermissions(ShowCtsRoleBasedPermissionsSchema):
+    """Parser for show cts role-based permissions"""
+
+    cli_command = 'show cts role-based permissions'
+
+    def cli(self, output=None):
+        if output is None:
+            out = self.device.execute(self.cli_command)
+        else:
+            out = output
+
+        cts_rb_permissions_dict = {}
+
+        # IPv4 Role-based permissions default:
+        rb_default_capture = re.compile(r"^IPv4\s+Role-based\s+permissions\s+(?P<default_group>default)")
+        # IPv4 Role-based permissions from group 42:Untrusted to group Unknown:
+        rb_permissions_capture = re.compile(
+            r"^IPv4\s+Role-based\s+permissions\s+from\s+group\s+(?P<src_grp_id>\d+):(?P<src_grp_name>\S+)\s+to\s+group\s((?P<unknown_group>Unknown)|(?P<dst_group_id>\d+):(?P<dst_group_name>\S+)):")
+        #         Deny IP-00
+        policy_action_capture = re.compile(r"^(?P<action_policy>(Permit|Deny))\s+(?P<action_policy_group>\S+)")
+        # ACCESS-01
+        policy_group_capture = re.compile(r"^(?P<policy_group>\w+-\d+)")
+        # RBACL Monitor All for Dynamic Policies : FALSE
+        monitor_dynamic_capture = re.compile(
+            r"^RBACL\s+Monitor\s+All\s+for\s+Dynamic\s+Policies\s+:\s+(?P<monitor_dynamic>(TRUE|FALSE))")
+        #RBACL Monitor All for Configured Policies : FALSE
+        monitor_configured_capture = re.compile(
+            r"^RBACL\s+Monitor\s+All\s+for\s+Configured\s+Policies\s+:\s+(?P<monitor_configured>(TRUE|FALSE))")
+
+        remove_lines = ()
```

Comment:
![image](https://user-images.githubusercontent.com/34322875/93940112-ed820980-fcf9-11ea-8563-66f4198525ce.png)
added on behalf of @KamyarZiabari from https://github.com/CiscoTestAutomation/genieparser/pull/219

Reply:
Remove lines was used to remove delimiter lines and headers. However, since we are accounting for all lines I went ahead and removed that.

Reply:
it seems the line in question is gone.